### ISSUE TITLE
Predictions between cubist_rules and rule_fit are different when a formula includes log10

### ISSUE THREAD

**Issue by gglee4ai** [tidymodels/rules#5]
I am trying to apply rule_fit to example of Rules package 0.0.1.
Today, I found that the prediction results are different between cubist_rules and rule_fit when log10 function is included in the model formula.
Please check these results. 

```r
data(car_prices, package = "modeldata")
set.seed(9932)
car_split <- initial_split(car_prices)
car_train <- training(car_split)
car_test <- testing(car_split)

cubist_fit <-
  cubist_rules(neighbors = 9, committees = 9) %>% 
  set_engine("Cubist") %>% 
  fit(log10(Price) ~ ., data = car_train)

rulefit_fit <-
  rule_fit() %>% 
  set_mode("regression") %>%
  fit(log10(Price) ~ ., data = car_train)

res <- 
  bind_cols(
    predict(cubist_fit, new_data = car_train),
    predict(rulefit_fit, new_data = car_train)
  ) %>% 
  rename(cubist = .pred, rulefit = .pred1)
res

###   cubist rulefit
### 1   4.32  20214.
### 2   4.50  33250.
### 3   4.50  32423.
### 4   4.51  33244.
### 5   4.49  33563.
```

ps1: I hope that rule_fit tunning tutorial is attached in the documents.

**Comment by topepo** [comment]
Looks like an issue with `xrf`: 

``` r
library(xrf)
#> Warning: package 'xrf' was built under R version 3.6.2

range(mtcars$mpg)
#> [1] 10.4 33.9

set.seed(244)
mod <- xrf(log(mpg) ~ ., data = mtcars, family = "gaussian")
predict(mod, mtcars[1:5, -1])
#>                          1
#> Mazda RX4         20.82131
#> Mazda RX4 Wag     21.03468
#> Datsun 710        22.88032
#> Hornet 4 Drive    21.32391
#> Hornet Sportabout 18.62595
```

<sup>Created on 2020-05-26 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

In the meantime, can you use a recipe instead?

**Comment by topepo** [comment]
I just looked at `?xrf` and: 

> transformation of the response variable is not supported. 

I'll try to put in a PR to change that and update the man file with `rules`.

**Comment by gglee4ai** [comment]
I realized that the transformation of all_outcome() in recipes can not be transformed inversely yet (ex. BoxCox or YeoJohnson). So, I am waiting for the inverse transformation to be implemented.

**Comment by topepo** [comment]
Fixed in 7959c59
